mirdata/beatles.py

```diff
@@ -49,41 +49,50 @@ class Track(object):
         sections (SectionData): sections annotation
 
     """
+
     def __init__(self, track_id, data_home=None):
         if track_id not in INDEX:
-            raise ValueError(
-                '{} is not a valid track ID in Beatles'.format(track_id))
+            raise ValueError("{} is not a valid track ID in Beatles".format(track_id))
 
         self.track_id = track_id
 
         self._data_home = data_home
         self._track_paths = INDEX[track_id]
 
         self.audio_path = utils.get_local_path(
-            self._data_home, self._track_paths['audio'][0])
+            self._data_home, self._track_paths["audio"][0]
+        )
 
-        self.title = os.path.basename(
-            self._track_paths['sections'][0]).split('.')[0]
+        self.title = os.path.basename(self._track_paths["sections"][0]).split(".")[0]
+
+    def __repr__(self):
```

Comment:
Can you print all of the cached properties too, something like `beats=BeatData('start_times', 'end_times', 'sections')`

Reply:
Can do!